src/cli/hpr_cli_trace.erl

```diff
@@ -0,0 +1,85 @@
+%%%-------------------------------------------------------------------
+%% @doc pp_cli_trace
+%% @end
+%%%-------------------------------------------------------------------
+-module(hpr_cli_trace).
+
+-behavior(clique_handler).
+
+-include("hpr.hrl").
+
+-export([register_cli/0]).
+
+register_cli() ->
+    register_all_usage(),
+    register_all_cmds().
+
+register_all_usage() ->
+    lists:foreach(
+        fun(Args) -> apply(clique, register_usage, Args) end,
+        [trace_usage()]
+    ).
+
+register_all_cmds() ->
+    lists:foreach(
+        fun(Cmds) -> [apply(clique, register_command, Cmd) || Cmd <- Cmds] end,
+        [trace_cmd()]
+    ).
+
+%%--------------------------------------------------------------------
+%% Config
+%%--------------------------------------------------------------------
+
+trace_usage() ->
+    [
+        ["trace"],
```

Comment:
I just remembered this trace from blockchain-core.
https://github.com/helium/blockchain-core/blob/master/src/cli/blockchain_cli_trace.erl

```sh
hpr trace start app_eui=aabbcc dev_eui=ddeeff
```
And it dumps to stdout if a file isn't specified. Might be useful in the future.

Reply:
This one just creates the file by default in trance function